### ISSUE TITLE
Supported more  layers in future?

### ISSUE THREAD

**Issue by jjqcat** [Cr33zz/Neuro_#2]
great work! I like it ,more layers supported I look forward to.

**Comment by Cr33zz** [comment]
Thanks, do you have any particular layers in mind?

**Comment by jjqcat** [comment]
for example, these layers as
Add, Concatenate, Subtract, Multiply, Average, Maximum
AveragePooling1D/2D, GlobalAveragePooling1D/2D
Bidirectional, TimeDistributed, GRU, LSTM, CuDNNGRU, CuDNNLSTM
 SeparableConv2D, DepthwiseConv2D
Cropping1D/2D, ZeroPadding1D/2D
BatchNormalization, 
AlphaDropout, GaussianDropout, GaussianNoise
SpatialDropout1D, SpatialDropout2D, SpatialDropout3D
GlobalMaxPooling1D/2D
ELU, LeakyReLU, ReLU, SeLU, PReLU
Sigmoid, Softmax, Softplus, Tanh
UpSampling1D/2D
Reshape, Permute
Embedding

**Comment by jjqcat** [comment]
sorry, maybe most of them have been supported.

**Comment by Cr33zz** [comment]
Yeah, most of them are, I plan to add recurrent models support via LTSM, GRU at some point in the future as well as 4D operations support for audio/video or 3D GANs. However, I'm just a single man, working on it in spare time, so I have to choose my battles well :)
